server.go

```diff
@@ -483,11 +478,21 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 		subscribers: make(map[uint64]*preimageSubscriber),
 	}
 
-	_, currentHeight, err := s.cc.ChainIO.GetBestBlock()
+	currentHash, currentHeight, err := s.cc.ChainIO.GetBestBlock()
 	if err != nil {
 		return nil, err
 	}
 
+	// Create an invoice expiry watcher that will expire our own invoices
+	// once block before they trigger a force close.
+	expiryWatcher := invoices.NewInvoiceExpiryWatcher(
+		clock.NewDefaultClock(), lncfg.DefaultIncomingBroadcastDelta+1,
```

Comment:
Using `lncfg.DefaultIncomingBroadcastDelta+1` here so that we don't go to chain for our hodl invoices.

Wondering whether this should be exposed in config? We don't expose our broadcast deltas, so I wasn't sure whether it was necessary.